Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/workbench/RetrieveProjectDAO.java

```diff
@@ -86,6 +86,30 @@ public ProjectDTO getProjectDTO(String projectId,String userId) throws QuadrigaS
 	}
 
 
+	/**
+	 * Meghana
+	 */
+	@SuppressWarnings("unchecked")
+	@Override
+	public List<ProjectDTO> getAllProjectsDTOByAccessibility(String accessibility) throws QuadrigaStorageException 

```

Comment:
please read the style guide we use and adjust accordingly (you can leave the 6 spaces for now, but the rest should be adjusted):
https://diging.atlassian.net/wiki/pages/viewpage.action?pageId=8781898